### ISSUE TITLE
IAM Database authentication for RDS

### ISSUE THREAD

**Issue by emilva** [spring-cloud/spring-cloud-aws#256]
Hi,

are there any plans for implementing support for IAM authentication for RDS?

http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.html

**Comment by gauz09** [comment]
Almost one year and no response! Doesn't look like there are many people asking for it. This will certainly be a good feature as part of AWS support from Spring. Hopefully, somebody will shed some light on the plans for this.

**Comment by emilva** [comment]
@gauz09 yeah, I think very few people know about this feature. Even AWS launched Secret Manager where they promote that it can automatically rotate RDS passwords! Well, with IAM authentication you do not need to rotate passwords as the token expires every 15 minute  ¯\_(ツ)_/¯

**Comment by barrtrek** [comment]
This would be great!
Trying to connect to an Aurora Postgres instance from a Spring Boot app.  Was a little bummed to see no Spring way of doing that with IAM.  Haven't written custom connection logic in some time O.o

**Comment by Ramzus** [comment]
+1 !
Would be great to see such feature in spring-cloud-aws !
